go/state/flush_benchmark_test.go

```diff
@@ -0,0 +1,43 @@
+package state
+
+import (
+	"github.com/Fantom-foundation/Carmen/go/common"
+	"testing"
+)
+
+func BenchmarkFromBytesSerializer(b *testing.B) {
```

Comment:
we do have almost identical benchmark in this PR: https://github.com/Fantom-foundation/Carmen/pull/842/files  

could this be used instead of creating a new one?